apps/rock-paper-scissors/src/utils/form.ts

```diff
@@ -0,0 +1,35 @@
+import { useState } from 'react'
+
+const isExists = (value: string) => (!value ? 'Field is required' : null);
+
+const isMinValue = (value: number, minDefaultValue: number) => (value < minDefaultValue ? `Minimal value ${minDefaultValue}` : null)
+
+const useForm = () => {
+  const [form, setForm] = useState('');
+  const routes = [
+    'create',
+    'lobby admin',
+    'detail',
+    'detail admin',
+    'join',
+    'Join game',
+    'game',
+    'move',
+    'reveal',
+    'round result',
+  ]
+
+  const openCreateForm = () => setForm('create');
+  const openLobbyForm = () => setForm('lobby');
+  const openDetailsForm = () => setForm('detail')
+
+  const closeForm = () => setForm('');
+  const onClickRouteChange = (param: string): void => {
+    const result = routes.filter(route => route === param)
+    setForm(result[0])
```

Comment:
What's the reason of mapping array if you can just call `setForm` with needed value?

Reply:
fixed